api/_lib/FanoutGraphqlExpressServerTest.test.ts

```diff
@@ -13,11 +13,11 @@ import {
   IStoredPubSubSubscription,
   MapSimpleTable,
 } from "fanout-graphql-tools";
-import * as http from "http";
-import * as killable from "killable";
+import http from "http";
+import killable from "killable";
 import { AddressInfo } from "net";
-import * as url from "url";
-import * as WebSocket from "ws";
+import url from "url";
+import WebSocket from "ws";
```

Comment:
Travis is failing with errors about WebSocket implementation. https://travis-ci.org/fanout/apollo-serverless-demo/builds/570945732#L396

It's not obvoius why, but if I had to guess it would be due to this change. iirc importing this from ws has the side-effect of defining global.WebSocket, which apollo later looks for. Gross but that's how it is.

Was it fully necessary to work on now to disable `allowSyntheticDefaultImports`, enable `esModuleInterop`, and change all these imports?

Reply:
@gobengo Wow, great find! 👍 

I have never seen TypeScript fail in this way before. Setting `esModuleInterop` should make `import * as foo` and `import foo` act identical for CJS modules 🤷‍♂ 

I made the change and the tests pass, thanks!